validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -320,6 +320,8 @@ def __init__(self, database, state_delta_store):
 
         self._address_regex = re.compile('^[0-9a-f]{70}$')
 
+        self._namespace_regex = re.compile('^[0-9a-f]{0,70}$')
```

Comment:
Even with the STL-149 changes, a byte will still be the atomic unit of address pathing. There will be cases where more than one byte is in the pointer, but never less than one byte. My preference is for enforcing an even number of characters.